user-management-and-permissions_5c07ccc604286304a71d0252.md

```diff
@@ -33,11 +35,57 @@ Owners can:
   projects in the organization will be revoked. Please note that this won't
   change their permissions on GitHub.
 
+- **Promote members to admins**.
+
 #### Billing management
 
 Owner can see the billing section and perform changes, such as enter credit
-card information, see the current spending, and edit invoice related
-information.
+card information, and see the current spending.
+
+#### Organization settings
+
+Owner can change the name of the organization. Note that this will not change
+the ID and unique URL of the organization.
+
+## Admin
+
+Admins have the same rights as the owner, and there can be multiple admins.
```

Comment:
But they can't delete an organization and request transferring ownership, right?

Reply:
It's about what you can do via UI. We will update this page when we introduce delete org/transferring ownership into UI.